test/tools/libtesteth/Options.cpp

```diff
@@ -72,6 +74,32 @@ void printVersion()
 {
     cout << prepareVersionString() << "\n";
 }
+}
+
+void Options::setVerbosity(int _level)
+{
+    static std::ostringstream strCout;  // static string to redirect logs to
+    static dev::LoggingOptions logOptions;
+    if (_level > verbosity)
+    {
+        verbosity = _level;
+        logOptions.verbosity = verbosity;
+        if (_level <= 0)
+        {
+            logOptions.verbosity = -1;
+            dev::setupLogging(logOptions);
```

Comment:
Now you can call this just once in the end of the function, instead of in each branch